docker-compose.yml

```diff
@@ -54,10 +54,8 @@ services:
       - /var/cache/open-apparel-registry-node-modules:/usr/local/src/node_modules
       - ./src/django/static:/usr/local/src/build
       - $HOME/.aws:/root/.aws:ro
-    env_file: .env.frontend
     environment:
       - REACT_APP_GIT_COMMIT=${REACT_APP_GIT_COMMIT:-latest}
-      - REACT_APP_API_URL=http://localhost:8081
```

Comment:
It looks like this was only used by the NodeJS API. I believe everything is hardcoded in the frontend:

https://github.com/open-apparel-registry/open-apparel-registry/blob/1c1182536baae9d22786ad0ef50f67b4a799f1d6/src/app/src/setupProxy.js#L18